### PR TITLE
Added Ratings and Snackbar

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Snackbar
Screenshot 2022-02-13 232805
RAtings
Screenshot 2022-02-13 232524

### CODE REVIEW

File: Components-JS/modal.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const doAction = document.querySelector('#modal-opener');
+const disp = document.querySelector('.modal-box')
+const container = document.querySelector(".container")
+
+//  function lateRun(){
```
Comment (sanjitsarkar) [inline Components-JS/modal.js]: remove unused codes

File: ComponentsCss/code.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    /* max-width: 75vw; */
+    background: #eee;
+    padding: 1.5rem 1rem;
+    border-radius: 10px;
+    margin-top: 1rem;
+    
+    overflow-y: scroll;
+}
+.code-box .code-head{
+    padding: 0.5rem;
```
Comment (sanjitsarkar) [inline ComponentsCss/code.css]: use px for smaller than 10px

**Review by sanjitsarkar — APPROVED** [review]

**Comment by Hamza-777** [comment]
Looks good hrishikesh.

**Review by Hamza-777 — APPROVED** [review]

File: Components-JS/modal.js (hunk 1 above)
Comment (md-amir-gauhar) [inline Components-JS/modal.js]: remove unwanted code

File: ComponentsCss/code.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+.code-box>.code>.line>span {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
+    white-space: pre;
+}
+.element {
+    color: #2f6f9f;
+}
+.attribute {
+    color: #006ee0;
```
Comment (md-amir-gauhar) [inline ComponentsCss/code.css]: use variables for colors

File: SnackBar/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    margin: 1em;
+    padding: 1em;
+    border-radius: 10px;
+    text-decoration: none;
+    font-size: larger;
+    
+}
+.snackbar-btn button:active{
+    color: white;
+    background-color: #9552f3;
```
Comment (md-amir-gauhar) [inline SnackBar/snackbar.css:23]: use variables again

File: SnackBar/snackbar.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+            &lt;p&gt;Cant send photo.Retry in 5 seconds&lt;/p&gt;
+          </span></span> 
+          &lt;/<span class="hljs-built_in">div</span>&gt;
+          &lt;<span class="hljs-built_in">div</span> class=<span class="hljs-string">"retry-btn"</span>&gt;<span class="hljs-string">
+            &lt;<span class="hljs-built_in">button</span>&gt;<span class="hljs-string"> Retry</span> &lt;/<span class="hljs-built_in">button</span>&gt;
+          &lt;<span class="hljs-built_in">div</span> class=<span class="hljs-string">"close-btn"</span>&gt;<span class="hljs-string">
+            &lt;<span class="hljs-built_in">button</span>&gt;<span class="hljs-string">Btn-icon</span> &lt;/<span class="hljs-built_in">button</span>&gt;
+          </span></span> &lt;/<span class="hljs-built_in">div</span>&gt; 
+        </span> &lt;/<span class="hljs-built_in">div</span>&gt; 
+      </span> &lt;/<span class="hljs-built_in">div</span>&gt;
```
Comment (md-amir-gauhar) [inline SnackBar/snackbar.html:94]: use some code formatter. e.g prettier

**Review by md-amir-gauhar — APPROVED** [review]

**Pull request merged**